src/Elcodi/Store/PaymentBridgeBundle/Services/PaymentBridge.php

```diff
@@ -172,9 +172,13 @@ public function getExtraData()
                     ->getPurchasable()
                     ->getName();
 
+                /*
+                 * Elcodi stores amounts in integer cents (10USD = 1000)
+                 * but Paypal expects decimal (10.00USD)
+                 */
                 $orderLineArray['amount'] = $orderLine
                     ->getAmount()
-                    ->getAmount();
+                    ->getAmount() / 100;
```

Comment:
Yes, the first is the `Money` instance (accessed via `Elcodi\Component\Cart\Entity\Interfaces\PriceInterface#getAmount()`) and the other is the `Money::amount` property mapped in `Elcodi\Component\Cart\Entity\OrderLine`). 

1) I get the `Money`
2) I get the `amount` property of the money object

RFC about renaming one of them?